BankSDK/GiniBankSDK/Documentation/source/Migration to version 3.x.x.md

```diff
@@ -316,34 +316,45 @@ If you need to animate the illustrations on the onboarding pages implement the `
 
 ## Digital Invoice Overview Screen
 
+The new digital invoice overview screen uses the global UI customization options. You can discard the old screen specific customizations.
+
+TODO You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniBankSDK/3.0.0-beta07/customization-guide.html#digital-invoice-screen).
+
 ### Breaking Changes
 
-Info box was removed.
+We removed the ability for users to manually add additional articles.
+We also removed the info box.
 
 ### New Features
 
-#### New UI
+#### Bottom navigation bar
+
+You can show a bottom navigation bar by passing true to `GiniBankConfiguration.shared.bottomNavigationBarEnabled`. There is a default implementation, but you can also use
+your own by implementing the `DigitalInvoiceNavigationBarBottomAdapter` interface and passing it to `GiniBankConfiguration.shared.digitalInvoiceNavigationBarBottomAdapter`.
+
+## Digital Invoice Edit Article screen
+
+The new edit article screen uses the global UI customization options and is presented as a bottom sheet on phones and as a dialog on iPad. You can discard the old screen specific customizations.
 
-The new digital invoice screen uses the global UI customization options. You can discard the old screen specific customizations.
+[Here](TODO) you can find the detailed description on how to customize this screen.
 
-You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniBankSDK/3.0.0-beta06/customization-guide.html#digital-invoice-screen).
+### Breaking Changes
 
-## TODO Edit line item screen
+We removed the ability to deselect the article from this screen.
 
 ### New Features
 
-#### New UI
+Easier currency and amount input when editing an article.
 
-The new edit line item screen uses the global UI customization options. You can discard the old screen specific customizations.
+## Digital Invoice Help screen
 
-You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniBankSDK/3.0.0-beta06/customization-guide.html#edit-line-item-screen).
+The new help screen uses the global UI customization options. You can discard the old screen specific customizations.
 
-## TODO Help screen (former Info screen)
+You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniBankSDK/3.0.0-beta07/customization-guide.html#return-assistant-help-screen) and [here](https://www.figma.com/file/623Fahuwq8tUzSWQPd23Ie/iOS-Gini-Bank-SDK-3.0.0-UI-Customisation?node-id=1356%3A11409&t=hzHQDEsTJv9ZVE7M-1).
```

Comment:
[Suggestion] I think the first link should be updated to point to `digital-inboice-help-screen`. Also I would not add the figma link here again. It is already available in the customization guide.
```suggestion
You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniBankSDK/3.0.0-beta07/customization-guide.html#digital-invoice-help-screen).
```